### ISSUE TITLE
Add setting option to "bypass" query string parameters

### ISSUE THREAD

**Issue by jobee** [Flowpack/Flowpack.FullPageCache#7]
Skipping the fullpage cache for requests that include a query string by default, is probably a good decision.
However, adding this feature would enable cached documents even for query string parameters that typically won't change the cached HTML response.

Example configuration in a Neos project would look like...

```yaml
Flowpack:
  FullPageCache:
    bypassQueryStringParameters:
      - utm_source
      - utm_medium
```

I would give it a try and submit a PR, if there are no objections.

**Comment by Sebobo** [comment]
That sounds great!

**Comment by mficzel** [comment]
Is resolved with version 2.0.0 via setting `setting. queryParams.allow`
